codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Error.java

```diff
@@ -46,4 +61,8 @@
     public long code() {
         return code;
     }
+
+    public static Http2Error valueOf(long value) {
```

Comment:
https://tools.ietf.org/html/rfc7540#section-6.4 
> The RST_STREAM frame contains a single unsigned, 32-bit integer
   identifying the error code (Section 7).

We represent this with a `long`.